### PR TITLE
Refactored NavBar in  Typescript

### PR DESCRIPTION
**Pull request by sandrananceva**
## OVERVIEW

Refactored NavBar in Typescript

## WHERE SHOULD THE REVIEWER START?

`src/components/NavBar/index.tsx`

## CHECKLIST

_Be sure all items are_ ✅ _before submitting a PR for review._

- [ ] Verify the linters and tests pass: `yarn review`
- [ ] Verify you bumped the lib version according to [Semantic Versioning Standards](http://semver.org)
- [ ] Verify you updated the CHANGELOG
- [ ] Verify this branch is rebased with the latest master

### CODE REVIEW

File: src/components/NavBar/NavBar.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+// @flow
```
Comment (ljubomirsk) [inline src/components/NavBar/NavBar.tsx]: Remove this annotation everywhere

File: src/components/NavBar/styled.tsx [hunk 2]
```diff
@@ -14,7 +18,7 @@ export const NavbarContainer = styled.div`
   overflow: hidden;
 `;
 
-export const NavbarLogo = styled.img`
+export const NavbarLogo = styled.img<{ src?: any }>`
```
Comment (ljubomirsk) [inline src/components/NavBar/styled.tsx]: You shouldn't need to specify src as a prop since the `` tag accepts that prop.

**Review by ljubomirsk — CHANGES_REQUESTED** [review]

File: src/components/NavBar/NavBar.tsx (hunk 1 above)
Comment (sandrananceva) [reply src/components/NavBar/NavBar.tsx]: Solved

File: src/components/NavBar/styled.tsx (hunk 2 above)
Comment (sandrananceva) [reply src/components/NavBar/styled.tsx]: Solved

**Pull request closed without merging**